### ISSUE TITLE
Generate helm-docs readme for all charts

### ISSUE THREAD

**Issue by Allex1** [open-telemetry/opentelemetry-helm-charts#430]
@TylerHelmuth @dmitryax I was thinking to add [helm-docs](https://github.com/norwoodj/helm-docs) for all of our chart like [Grafana charts](https://github.com/grafana/helm-charts/tree/main/charts/grafana#configuration) have for example. We use them in all charts and I find it very useful. I can also add contributing instructions for users to keep them in sync when they make changes in values.yaml. WDYT ?

**Comment by dmitryax** [comment]
Sounds good to me 👍

**Comment by TylerHelmuth** [comment]
Agreed. Id say we should add something to the workflows that enforces things stay in sync, like we do with examples.

**Comment by Saki042** [comment]
@TylerHelmuth , Can you confirm whether this enhancement requirement is still open ?

**Comment by TylerHelmuth** [comment]
Yes this would still be helpful

**Comment by Saki042** [comment]
@Allex1 / @TylerHelmuth , Can you elaborate more on the issue and I am new to open source so it would be really helpful if you provide me some guide regarding changes. 
Can you please assign it to me?

**Comment by marcschaeferger** [comment]
I’d be happy to take this on if it’s still available.

I’ve been using `helm-docs` for quite some time and already use it for two open-source Helm charts as well as numerous private Helm charts. I’m therefore familiar with generating and maintaining chart documentation from `values.yaml`, including keeping the generated README content in sync via CI.

**Comment by TylerHelmuth** [comment]
@marcschaeferger I think this feature would still be useful.

**Comment by marcschaeferger** [comment]
I'll prepare something today.

**Comment by TylerHelmuth** [comment]
@marcschaeferger please start with 1 chart.
